pages/contact.vue

```diff
@@ -1,15 +1,109 @@
 <template>
-  <div class="white-wrapper">
-    Contact Page
+  <div class="background-section">
+    <div class="title">
+      <h1 class="cursive">Contact & Directions</h1>
+      <h4><font-awesome-icon icon="phone"/> (617) 924-9595</h4>
+    </div>
+    <div class="contact-section">
+        <div class="outside-image">
+          <img src="outside2.jpg">
+        </div>
+        <div class="store-info-section">
+          <contact-info />
+        </div>
+    </div>
+    <div class="call-section">
+      <div class="order">
+        <a :href="`tel:${6179249595}`">
+          <div class='phone-number'> 
+            <font-awesome-icon icon="phone"/> (617) 924-9595 
+          </div>
+        </a>
+      </div>
+      <a :href="`tel:${6179249595}`">
+        <button class="call-button">Call Now</button>
+      </a>
+    </div>
   </div>
 </template>
 
 <script>
+import ContactInfo from '../components/ContactInfo'
+
 export default {
 
```

Comment:
This works as is but it is nuxt convention to add the `components: { ContactInfo }` portion to the default export.

Reply:
Made correction to follow nuxt convention.